internal/manifest/version.go

```diff
@@ -74,6 +74,8 @@ type TableStats struct {
 	ValueBlocksSize uint64
 	// CompressionType is the compression type of the table.
 	CompressionType sstable.Compression
+	// NumTombstoneDenseBlocks is the number of tombstone-dense data blocks in this table.
```

Comment:
We should define what tombstone-dense means here, likely in terms of the constants defined in internal/base or in options.